error: constexpr variable cannot have non-literal type 'const std::array<BoutReal, size>' (aka 'const array<double, size>')

```
/usr/ports/science/bout++/work/BOUT-dev-4.4.2/src/solver/impls/adams_bashforth/adams_bashforth.cxx:57:40: error: constexpr variable cannot have non-literal type 'const std::array<BoutReal, size>' (aka 'const array<double, size>')
  constexpr std::array<BoutReal, size> facs{989.0 * fac,   5888.0 * fac,  -928.0 * fac,
                                       ^
/usr/ports/science/bout++/work/BOUT-dev-4.4.2/src/solver/impls/adams_bashforth/adams_bashforth.cxx:57:40: error: implicit instantiation of undefined template 'std::array<double, 9>'
/usr/include/c++/v1/__tuple:219:64: note: template is declared here
template <class _Tp, size_t _Size> struct _LIBCPP_TEMPLATE_VIS array;
                                                               ^
/usr/ports/science/bout++/work/BOUT-dev-4.4.2/src/solver/impls/adams_bashforth/adams_bashforth.cxx:70:40: error: constexpr variable cannot have non-literal type 'const std::array<BoutReal, size>' (aka 'const array<double, size>')
  constexpr std::array<BoutReal, size> facs{751.0 * fac,  3577.0 * fac, 1323.0 * fac,
                                       ^
/usr/ports/science/bout++/work/BOUT-dev-4.4.2/src/solver/impls/adams_bashforth/adams_bashforth.cxx:70:40: error: implicit instantiation of undefined template 'std::array<double, 8>'
/usr/include/c++/v1/__tuple:219:64: note: template is declared here
template <class _Tp, size_t _Size> struct _LIBCPP_TEMPLATE_VIS array;
                                                               ^
/usr/ports/science/bout++/work/BOUT-dev-4.4.2/src/solver/impls/adams_bashforth/adams_bashforth.cxx:83:40: error: constexpr variable cannot have non-literal type 'const std::array<BoutReal, size>' (aka 'const array<double, size>')
  constexpr std::array<BoutReal, size> facs{41.0 * fac,  216.0 * fac, 27.0 * fac,
                                       ^
/usr/ports/science/bout++/work/BOUT-dev-4.4.2/src/solver/impls/adams_bashforth/adams_bashforth.cxx:83:40: error: implicit instantiation of undefined template 'std::array<double, 7>'
/usr/include/c++/v1/__tuple:219:64: note: template is declared here
template <class _Tp, size_t _Size> struct _LIBCPP_TEMPLATE_VIS array;
                                                               ^
/usr/ports/science/bout++/work/BOUT-dev-4.4.2/src/solver/impls/adams_bashforth/adams_bashforth.cxx:96:40: error: constexpr variable cannot have non-literal type 'const std::array<BoutReal, size>' (aka 'const array<double, size>')
  constexpr std::array<BoutReal, size> facs{19.0 * fac, 75.0 * fac, 50.0 * fac,
                                       ^
/usr/ports/science/bout++/work/BOUT-dev-4.4.2/src/solver/impls/adams_bashforth/adams_bashforth.cxx:96:40: error: implicit instantiation of undefined template 'std::array<double, 6>'
/usr/include/c++/v1/__tuple:219:64: note: template is declared here
template <class _Tp, size_t _Size> struct _LIBCPP_TEMPLATE_VIS array;
                                                               ^
/usr/ports/science/bout++/work/BOUT-dev-4.4.2/src/solver/impls/adams_bashforth/adams_bashforth.cxx:108:40: error: constexpr variable cannot have non-literal type 'const std::array<BoutReal, size>' (aka 'const array<double, size>')
  constexpr std::array<BoutReal, size> facs{7.0 * fac, 32.0 * fac, 12.0 * fac, 32.0 * fac,
                                       ^
/usr/ports/science/bout++/work/BOUT-dev-4.4.2/src/solver/impls/adams_bashforth/adams_bashforth.cxx:108:40: error: implicit instantiation of undefined template 'std::array<double, 5>'
/usr/include/c++/v1/__tuple:219:64: note: template is declared here
template <class _Tp, size_t _Size> struct _LIBCPP_TEMPLATE_VIS array;
                                                               ^
/usr/ports/science/bout++/work/BOUT-dev-4.4.2/src/solver/impls/adams_bashforth/adams_bashforth.cxx:120:40: error: constexpr variable cannot have non-literal type 'const std::array<BoutReal, size>' (aka 'const array<double, size>')
  constexpr std::array<BoutReal, size> facs{1.0 * fac, 3.0 * fac, 3.0 * fac, 1.0 * fac};
                                       ^
```

Version: 4.4.2
clang-14
FreeBSD 13.1

--- a/src/solver/impls/adams_bashforth/adams_bashforth.cxx
+++ b/src/solver/impls/adams_bashforth/adams_bashforth.cxx
@@ -7,6 +7,8 @@
 
 #include <output.hxx>
 
+#include <array>
+
 namespace {
 BoutReal lagrange_at_position_denominator(const std::deque<BoutReal>& grid,
                                           const int position, const int order) {

